api/types.go

```diff
@@ -661,6 +640,19 @@ func (d *Duration) UnmarshalJSON(b []byte) (err error) {
 	return nil
 }
 
+// ErrorResponse implements a structured error interface that is returned from the Ollama server
+type ErrorResponse struct {
+	Err  string `json:"error,omitempty"` // The annotated error from the server, helps with debugging the code-path
+	Hint string `json:"hint,omitempty"`  // A user-friendly message about what went wrong, with suggested troubleshooting
```

Comment:
Comments should be in the line above

```suggestion
	// Err is the error from the server. It helps with debugging the code-path
	Err  string `json:"error"`

	// Hint is a user-friendly message about what went wrong, with suggested troubleshooting
	Hint string `json:"hint"`
```